ape_solidity/compiler.py

```diff
@@ -270,3 +270,36 @@ def load_dict(data: Union[str, dict]) -> Dict:
                 contract_types.append(ContractType.parse_obj(contract_type))
 
         return contract_types
+
+    def get_imports(
+        self, contract_filepaths: List[Path], base_path: Optional[Path]
+    ) -> Dict[str, List[str]]:
+        def import_str_to_paths(path: str, contract_filenames) -> List[str]:
+            path = path.replace("import", "").replace('"', "").replace(";", "").strip()
```

Comment:
```suggestion
            path = path.replace("import", "").replace('"', "").strip("; ")
```

was able to combine the effects with one less method, since `;` is always at the end and strip can take off more than one character by passing in a longer string to it